### DISCUSSION TITLE
Is possible to have multi version of the same framework?

### DISCUSSION THREAD

**Discussion by bmirandan** [SAP/luigi#2944] [Q&A]
I would like to know if for instance can i have an old front on vue 2, and starting doing a newer version with vue3.
Is this achievable with luigi?

**Comment by hardl** [comment]
Hi @bmirandan, this is possible, yes. Luigi micro frontends are completely independent and isolated. You can also make use of luigi's feature toggle mechanism to switch between the two versions (https://docs.luigi-project.io/docs/navigation-parameters-reference/?section=visibleforfeaturetoggles).
